modules/staging.nix

```diff
@@ -14,6 +14,8 @@ with import ./../lib.nix;
 
       # DEVOPS-64: disable log bursting
       journald.rateLimitBurst    = 0;
+
+      monitoring-exporters.graylogHost = "${config.deployment.arguments.globals.monitoringNV.name}-ip:5044";
```

Comment:
@johnalotoski, correct me if that is not what you suggested.

Reply:
@jbgi, this tests good.  It looks to me like the same single line addition needs to be made to testnet.nix and production.nix also for logging to work in those environments.  Additional conditional checks will be added shortly in another PR.